packages/ipfs/src/cli/commands/block/put.js

```diff
@@ -2,6 +2,8 @@
 
 const fs = require('fs')
 const multibase = require('multibase')
+/** @type {typeof import('it-concat').default} */
+// @ts-ignore - it-contact has invalid type defs for commonjs
```

Comment:
Why not make changes to `it-concat` directly? 
https://github.com/alanshaw/it-concat/pull/4

Reply:
Once that propagates, we can remove this. I just don't want to block on the other packages.